docs/setup/hooks.md

```diff
@@ -16,14 +15,25 @@ redirect_from: ../hooks.html
 Like other version control systems, lakeFS allows the configuration of `Actions` to trigger when predefined events occur.
 
 Supported Events:
-1. `pre_commit` - Action runs when the commit occurs, before the commit is finalized.
-1. `post_commit` - Action runs after the commit is finalized.
-1. `pre_merge` - Action runs when the merge occurs, before the merge is finalized.
-1. `post_merge` - Action runs after the merge is finalized.
+
+| Event                | Description                                                                    |
+|----------------------|--------------------------------------------------------------------------------|
+| `pre_commit`         | Runs when the commit occurs, before the commit is finalized                    |
+| `post_commit`        | Runs after the commit is finalized                                             |
+| `pre_merge`          | Runs on the source branch when the merge occurs, before the merge is finalized |
+| `post_merge`         | Runs on the merge result, after the merge is finalized                         |
+| `pre_create_branch`  | Runs on the source branch prior to creating a new branch                       |
+| `post_create_branch` | Runs on the new branch after the branch was created                            |
+| `pre_delete_branch`  | Runs prior to deleting a branch                                                |
+| `post_delete_branch` | Runs after the branch was deleted                                              |
+| `pre_create_tag`     | Runs prior to creating a new tag                                               |
+| `post_create_tag`    | Runs after the tag was created                                                 |
+| `pre_delete_tag`     | Runs prior to deleting a tag                                                   |
+| `post_delete_tag`    | Runs after the tag was deleted                                                 |
 
 lakeFS `Actions` are handled per repository and cannot be shared between repositories.
 Failure of any `Hook` under any `Action` of a `pre_*` event will result in aborting the lakeFS operation that is taking place.
-On the contrary, `Hook` failures under any `Action` of a `post_*` event will not affect the same operation.
+`Hook` failures under any `Action` of a `post_*` event will not revert the operation.
```

Comment:
abort? or block the operation?

Reply:
I don't think abort or block is appropriate since the operation already happened. Maybe rollback?